Biometric authentication is not initializing

**Describe the bug**

Biometric authentication is not initializing on the latest Windows builds.

<img width="1203" height="289" alt="Image" src="https://github.com/user-attachments/assets/494274c7-ab07-4b3d-997b-a365a38aafd5" />


Console shows:

```
Error: Cannot find module 'C:\Users\Test\AppData\Local\Programs\Trezor Suite\resources\app.asar\dist\winHelloChildProcess.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1390:15)
    at defaultResolveImpl (node:internal/modules/cjs/loader:1032:19)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1037:22)
    at Module._load (node:internal/modules/cjs/loader:1199:37)
    at c._load (node:electron/js2c/node_init:2:17993)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:244:24)
    at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:186:5)
    at node:internal/main/run_main_module:36:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

**Info:**
 - Suite version: desktop 25.10.0 (6464476043e0d9ef7f9565561c7b4511974aaba4)
 - Browser: Trezor Suite 25.10.0
 - OS: Win32
 - Screen: 2558x1298
 - Transport: BridgeTransport 3.1.0

--- a/packages/suite-desktop-core/webpack/core.webpack.config.ts
+++ b/packages/suite-desktop-core/webpack/core.webpack.config.ts
@@ -53,6 +53,7 @@ const threads = sync(`${threadPath}/**/*.ts`).map(globMatch => {
 });
 
 // Add Windows Hello child process to the build
+const winHelloChildProcessKey = 'winHelloChildProcess'; // must match the expected /dist filename that WinHelloProcessManager requires
 const winHelloChildProcessPath = path.join(
     __dirname,
     '../../suite-desktop-native/src/winHelloChildProcess.ts',
@@ -76,12 +77,13 @@ const config: webpack.Configuration = {
     target: 'electron-main',
     mode: isDev ? 'development' : 'production',
     devtool: 'source-map',
+    // Note that the entries key is important, it sets the the output file name in dist/
     entry: [
         // NOTE: in DEV ONLY pick app with react dev tools installed
         { app: isDev ? 'app-with-devtools' : 'app' },
         { preload: 'preload' },
         ...threads.map(thread => ({ [String(thread)]: thread })),
-        { winHelloChildProcessPath },
+        { [winHelloChildProcessKey]: winHelloChildProcessPath },
     ].reduce(
         (prev, cur) => ({
             ...prev,
@@ -89,7 +91,7 @@ const config: webpack.Configuration = {
                 (acc, [key, value]) => ({
                     ...acc,
                     [key]:
-                        key === 'winHelloChildProcessPath'
+                        key === winHelloChildProcessKey
                             ? value
                             : path.resolve(__dirname, `../src/${value}.ts`),
                 }),

